extra/debian/control

```diff
@@ -59,7 +59,6 @@ Depends: ${shlibs:Depends},
          gstreamer1.0-plugins-base,
          gstreamer1.0-plugins-good,
          gstreamer1.0-tools,
-         gstreamer1.0-x,
```

Comment:
I don't agree with removing `xvimagesink` from the default stb-tester installation, considering how many people start by trying it out on their desktop. I see how it's unnecessary on a headless test rig, but we should explore other solutions to that (perhaps splitting out an `stb-tester-core` package with the minimal dependencies) in a different pull request.